docs/modules/record-encryption/proc-configuring-record-encryption-filter.adoc

```diff
@@ -0,0 +1,49 @@
+// file included in the following:
+//
+// assembly-record-encryption.adoc
+
+[id='proc-configuring-record-encryption-filter-{context}']
+= Setting up the Record Encryption filter
```

Comment:
I think this section should come before the KMS configuration information and should probably contain the KMS config sections.